### ISSUE TITLE
Fix accessibility issue on the Data and Performance Hub

### ISSUE THREAD

**Issue by katelunceford** [cityofaustin/atd-data-tech#27600]
The latest [Siteimprove report](https://my2.us.siteimprove.com/Accessibility/1067160/NextGen/IssuesOverview/1?conformance=0,1&pageSegments=&issueKind=1&difficulty=&elementType=&responsibility=&exceptTags=&includingTags=&tagFilterType=2) identifies one A issue with https://data.mobility.austin.gov/

The task for this issue is to fix the A issue and try to fix all other issues as well.

<img width="1114" height="557" alt="Image" src="https://github.com/user-attachments/assets/f4b60400-8590-4f23-8fda-9299e77d83da" />

We want an accessibility score of >90 (per this [video](https://cityofaustin.sharepoint.com/sites/COADigitalAccessibility/_layouts/15/stream.aspx?id=%2Fsites%2FCOADigitalAccessibility%2FDigital+Accessibility+Media%2FDACP_LunchNLearn_Siteimprove.mp4&referrer=StreamWebApp.Web&referrerScenario=AddressBarCopied.view.30024671-a626-402e-bfe1-0a29e2866538&startedResponseCatch=true) around 19:15 in) and our current score is 94.2, so we're already in good shape. :)

<img width="1117" height="603" alt="Image" src="https://github.com/user-attachments/assets/712016b8-a537-41a3-9f08-2f3886e865e7" />

## Background

The City must meet the Department of Justice's ADA Title II ruling deadline. The ruling states that all public-facing digital content and applications provided by local and state governments must meet [Web Content Accessibility Guidelines (WCAG)](https://cityofaustin.sharepoint.com/sites/COADigitalAccessibility/SitePages/Understanding-WCAG.aspx?xsdata=MDV8MDJ8RGlhbmEuTWFydGluQGF1c3RpbnRleGFzLmdvdnwyYmU2YjhhYmQwNjU0YzhlMmIzYjA4ZGUxMGM3NTBlZnw1YzVlMTlmNmE2YWI0YjQ1YjFkMGJlNDYwOGE5YTY3ZnwwfDB8NjM4OTY2NjQ1NTA0OTgyOTM2fFVua25vd258VFdGcGJHWnNiM2Q4ZXlKRmJYQjBlVTFoY0draU9uUnlkV1VzSWxZaU9pSXdMakF1TURBd01DSXNJbEFpT2lKWGFXNHpNaUlzSWtGT0lqb2lUV0ZwYkNJc0lsZFVJam95ZlE9PXwwfHx8&sdata=VCtuaG1RdHY0eWRyZVo4R1drVS9FazdHVko4WE41MjEzR0JnSm95aExIRT0%3d) 2.1 AA (and A) standards by April 24, 2026. [Read more details about the federal ruling here.](https://cityofaustin.sharepoint.com/sites/COADigitalAccessibility/SitePages/ADA-Title-II-Rule-Compliance.aspx?xsdata=MDV8MDJ8RGlhbmEuTWFydGluQGF1c3RpbnRleGFzLmdvdnwyYmU2YjhhYmQwNjU0YzhlMmIzYjA4ZGUxMGM3NTBlZnw1YzVlMTlmNmE2YWI0YjQ1YjFkMGJlNDYwOGE5YTY3ZnwwfDB8NjM4OTY2NjQ1NTA0OTk3MjM4fFVua25vd258VFdGcGJHWnNiM2Q4ZXlKRmJYQjBlVTFoY0draU9uUnlkV1VzSWxZaU9pSXdMakF1TURBd01DSXNJbEFpT2lKWGFXNHpNaUlzSWtGT0lqb2lUV0ZwYkNJc0lsZFVJam95ZlE9PXwwfHx8&sdata=TXBEMzZkUWpTaWY0MGIycVhheHBLZE82cTI0OWlSMjFuNjVwSE9LSCs4az0%3d)

<img width="1065" height="377" alt="Image" src="https://github.com/user-attachments/assets/84ed5011-9989-4e95-8292-b8e3e522c9b4" />

**Comment by amenity** [comment]
@tillyw - do you have a pull request for this going? I'm hoping to close out #26406 soon.

**Comment by amenity** [comment]
& of course, lmk if you need more info and/or we should bump up the estimate!

**Comment by tillyw** [comment]
I have linked my [pull request](https://github.com/cityofaustin/atd-data-and-performance/pull/444)! @amenity I was unable to sign in directly to SiteImprove using our credentials, but if you or Kate have access and are able to confirm, please do!
